src/app/onto-search/onto-search.component.scss

```diff
@@ -0,0 +1,13 @@
+.state-type {
+  float: right;
+}
+
+.state-labels {
+  white-space: normal;
+  line-height: 1em;
+}
+
+::ng-deep .state-suggestion {
+  background-color: lightblue !important;
```

Comment:
Is the !important necessary here?

Reply:
Yes, because it overwrites a mat component style.